### ISSUE TITLE
Test failure

### ISSUE THREAD

**Issue by fabaff** [zabuldon/teslajsonpy#121]
One test is failing. This came up during the review for the NixOS package.

```bash
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.8.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
rootdir: /build/source, configfile: tox.ini
collected 151 items                                                            

tests/test_tesla_exception.py ...............                            [  9%]
[...]
tests/unit_tests/homeassistant/test_vehicle_device.py ..Fssss            [100%]

=================================== FAILURES ===================================
_____________________________ test_values_on_init ______________________________

monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7ffff5e3e700>

    def test_values_on_init(monkeypatch):
        """Test values after initialization."""

        _mock = TeslaMock(monkeypatch)
        _controller = Controller(None)

        _data = _mock.data_request_vehicle()
        _device = VehicleDevice(_data, _controller)

        assert _device is not None

        assert _device.car_name() is not None
        assert _device.car_name() == "Nikola 2.0"

        assert _device.car_type is not None
        assert _device.car_type == "Model S"

        assert _device.car_version is not None
        assert _device.car_version == ""

        assert _device.id() is not None
        assert _device.id() == 12345678901234567

        assert _device.sentry_mode_available is not None
>       assert _device.sentry_mode_available
E       assert False
E        +  where False = <teslajsonpy.homeassistant.vehicle.VehicleDevice object at 0x7ffff5e3eb20>.sentry_mode_available

tests/unit_tests/homeassistant/test_vehicle_device.py:59: AssertionError
=========================== short test summary info ============================
FAILED tests/unit_tests/homeassistant/test_vehicle_device.py::test_values_on_init
=================== 1 failed, 60 passed, 90 skipped in 0.83s ===================
```

It would be nice if you could have a look. Thanks.

**Comment by alandtse** [comment]
As a note, I have no idea how to reproduce it. I'll leave this open in case someone else sees the issue.

**Comment by graham33** [comment]
I happened to notice this too, I think the problem is just that the mock vehicle data is not well isolated between tests and is sometimes modified.  I'll propose a fix in a PR.

**Comment by graham33** [comment]
The reason I (and I think @fabaff too) had this issue is that the nixpkgs expression omitted the pytest-asyncio dependency, meaning all the asyncio tests were skipped.  It turns out that some of the asyncio tests were leaving the shared test data in a particular state so that the cross-test pollution issue was masked when you ran all the tests, and only showed up when you ran a subset of tests.

I've put in a PR to fix this in nixpkgs, but see #155 which should hopefully resolve the cross-test dependency issues
